wcag-ruleset-review-process.md

```diff
@@ -1,77 +1,89 @@
 
-## Public Feedback
-
-Public feedback received on the WCAG Ruleset is processed by the ACT Task Force. Questions or issues can either be submitted directly into the ACT Task Force GitHub repository, or be copied into GitHub issues from questions sent to the ACT public mailing list.
+1. If AG approves the changes, the rule is republished on the WAI website with the "proposal" indicator removed (i.e. it is published as an approved rule).
 
-Questions the ACT Task Force can answer will be responded to directly. Questions that may require updates to a rule will be forwarded to the rule provider, including a proposal from the ACT Task Force on how to resolve the question. If the rule provider uses the proposal without additional changes, the rule will be updated without having to go through the review process. The rule provider should still apply their own review process to the change. If a different solution is taken, or additional changes are made, the update will have to go through the review process.
+2. If AG requests editorial changes, a [drafts proposal](#1-draft-proposal) is created. Once merged, the updated rule is republished on the WAI website with the "proposal" indicator removed. This skips the implementations or validation steps.
 
-For the ACT Task Force agenda, public feedback will be prioritised over review of proposed rules. This ensures responsiveness to the wider community. The ACT Task Force should respond to questions within 5 work days. This may be a reply in which they defer to the rule provider for a solution.
-
-## Annual Review
+3. If AG requests non-editorial, the liaison is tasked to [drafts proposal](#1-draft-proposal).
 
-Since technologies and standards frequently change, rules require regular maintenance. To ensure quality over time, the ACT Task Force will review every rule in the WCAG Ruleset at least once a year, to ensure it is still current and correct. If any changes are deemed necessary, the ACT Task Force proposes an update to the rule to the original rule provider. If the change is taken up without further updates, the update is accepted without review.
+## Public Feedback
 
-Every time a rule is surveyed, it is checked to make sure it is up to date. This survey counts as an annual review, so that only rules that have not been surveyed in 12 months need to go through a separate review. If a rule has not been reviewed for 12 months, an ACT facilitator opens an issue for it, and assigns it to that rule's liaison. The liason has 10 work days to review if the rule needs to be updated. If the rule needs to be updated, the liaison has another 10 work days to propose any necessary changes to the rule provider.
+Public feedback received on the WCAG Ruleset is processed by the ACT Task Force. Questions or issues can either be submitted directly into the ACT Task Force GitHub repository, or be copied into GitHub issues from questions sent to the ACT public mailing list.
```

Comment:
For the WCAG ruleset, I guess we'll also be getting feedback sent to the AG or the CG, rather than the TF (typically, as issues on their respective Github repos). Do we need to duplicate these issues on the TF repo?
I'm notably concerned with the CG repo where there is next to no triaging happening on the issues, and where we might end up just fixing the issues rather than moving them to the TF repo (which requires digging out to remember if a rule is a WCAG published one or not).

Reply:
I don't quite know the practical implications yet. We should avoid duplicate issues, so we're going to need to funnel everything to the same place. We might just end up with a shared repo between TF / CG for rules and definitions.